aspnetcore/blazor/globalization-localization.md

```diff
@@ -97,63 +220,271 @@ By default, the Intermediate Language (IL) Linker configuration for Blazor WebAs
 
 ::: moniker-end
 
-While the culture that Blazor selects by default might be sufficient for most users, consider offering a way for users to specify their preferred locale. For a Blazor WebAssembly sample app with a culture picker, see the [`LocSample`](https://github.com/pranavkm/LocSample) localization sample app.
+::: zone-end
 
-### Blazor Server
+::: zone pivot="server"
 
-Blazor Server apps are localized using [Localization Middleware](xref:fundamentals/localization#localization-middleware). The middleware selects the appropriate culture for users requesting resources from the app.
+Blazor Server apps are localized using [Localization Middleware](xref:fundamentals/localization#localization-middleware). Add localization services to the app with <xref:Microsoft.Extensions.DependencyInjection.LocalizationServiceCollectionExtensions.AddLocalization%2A>.
 
-The culture can be set using one of the following approaches:
+In `Startup.ConfigureServices` (`Startup.cs`):
 
-* [Cookies](#cookies)
-* [Provide UI to choose the culture](#provide-ui-to-choose-the-culture)
+```csharp
+services.AddLocalization();
+```
 
-For more information and examples, see <xref:fundamentals/localization>.
+Specify the static culture in `Startup.Configure` (`Startup.cs`) immediately after Routing Middleware is added to the processing pipeline. The following example configures United States English:
 
-#### Cookies
+```csharp
+app.UseRequestLocalization(
```

Comment:
You can use `app.UseRequestLocalization("en-US")` for simplicity instead

Reply:
Perfect, thanks. I'll get this updated on the next commit.